edx_proctoring/api.py

```diff
@@ -640,9 +643,72 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
         exam_attempt_obj.started_at = datetime.now(pytz.UTC)
         exam_attempt_obj.save()
 
+    # email will be send when the exam is proctored and not practice exam
+    # and the status is verified, submitted or rejected
+    if exam_attempt_obj.taking_as_proctored and not exam_attempt_obj.is_sample_attempt and\
+            ProctoredExamStudentAttemptStatus.needs_status_change_email(exam_attempt_obj.status):
```

Comment:
Typically, as a pattern, for long conditional statements, in order to make a bit more readable and to avoid the \ character between line wraps, I do the conditional evaluation into a local variable - with parenthesis - right above the if statement like this:

should_send_status_email = (
   exam_attempt_obj.taking_as_proctored and
  not exam_attempt_obj.is_sample_attempt and
 ProctoredExamStudentAttemptStatus.needs_status_change_email(exam_attempt_obj.status)
)
if should_send_status_email:
   :